extensions/wikia/Discussions/DiscussionsHooksHelper.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+class DiscussionsHooksHelper {
+	/**
+	 * IRIS-5184: Exclude outgoing links in Forum content from Special:WhatLinksHere
+	 * @see SpecialWhatLinksHere::showIndirectLinks()
+	 *
+	 * @param array $pageLinkCondition
+	 */
+	public static function onSpecialWhatLinksHereBeforeQuery( array &$pageLinkCondition ) {
+		$pageLinkCondition[] = "page_namespace != '" . NS_WIKIA_FORUM_BOARD_THREAD . "'";
+	}
+
+	/**
+	 * IRIS-5184: Exclude outgoing links in Forum content from Special:WantedPages report
+	 * @see WantedPagesPage::getQueryInfo()
+	 *
+	 * @param IContextSource $context
+	 * @param array $queryInfo
+	 */
+	public static function onWantedPagesGetQueryInfo( IContextSource $context, array &$queryInfo ) {
+		$queryInfo['conds'][] = "pg2.page_namespace != '" . NS_WIKIA_FORUM_BOARD_THREAD . "'";
+	}
+
+	/**
+	 * @param array $vars JS variables to be added at the bottom of the page
+	 * @param $scripts
+	 */
+	public static function addDiscussionJsVariable( array &$vars, &$scripts ) {
+		global $wgDiscussionsApiUrl;
+		$vars['wgDiscussionsApiUrl'] = $wgDiscussionsApiUrl;
+	}
```

Comment:
Shouldn't this `return true;`?

Reply:
Not needed anymore since we backported upstream improvements in https://github.com/Wikia/app/pull/13538